pkg/proxy/userspace/proxier.go

```diff
@@ -131,6 +131,8 @@ func createProxier(loadBalancer LoadBalancer, listenIP net.IP, iptables iptables
 	if proxyPorts == nil {
 		proxyPorts = newPortAllocator(util.PortRange{})
 	}
+	glog.V(2).Info("Tearing down pure-iptables proxy rules.")
```

Comment:
Just once on startup. The log level here mimics the one in the iptables
Proxier, so if we change this we should change both.

On Wed, Aug 12, 2015 at 5:12 PM, Mike Danese <notifications@github.com>
wrote:

> In pkg/proxy/userspace/proxier.go
> <https://github.com/kubernetes/kubernetes/pull/12597#discussion_r36914412>
> :
>
> > @@ -131,6 +131,8 @@ func createProxier(loadBalancer LoadBalancer, listenIP net.IP, iptables iptables
> >  	if proxyPorts == nil {
> >  		proxyPorts = newPortAllocator(util.PortRange{})
> >  	}
> > +	glog.V(2).Info("Tearing down pure-iptables proxy rules.")
>
> how often will this be called? I would suggest a higher log level
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/12597/files#r36914412>.
>